client/src/App.js

```diff
@@ -33,12 +35,23 @@ function App() {
         getCards(setCells);
     }, []);
 
+    async function handleCardClick(index, ) {
+        const response = await fetch(`http://127.0.0.1:8001/api/guess?index=${index}`)
+        const jsonData = await response.json();
+        console.log(jsonData);
+        if (jsonData === "blue has won! play again?" || jsonData === "blue has won! play again?") {
```

Comment:
hmm, if backend changes the message, this code breaks. So maybe have backend send a formal response so that our code doesn't break?